test/helper/passwordHelpers.test.js

```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
+const { expect } = chai;
+const { checkPasswordStrength, passwordsMatch } = require('../../src/helper/passwordHelpers');
+
+describe('passwordHelpers tests', function () {
+	describe('checking validity of password strength', function () {
+		it('should return false if the passwords do NOT match the requirements', function () {
+			const someWeekPasswords = ['somepassword', 'somepassword1', 'somepassword1!', 'somepassword1!@#', 'afsf@#$'];
+			someWeekPasswords.forEach((weakPassword) => {
+				expect(checkPasswordStrength(weakPassword))
+					.to.equal(false);
+			});
+		});
+		it('should return true if the passwords match the requirements', function () {
+			const someWeekPasswords = ['somepassworD4!', 'somepassworD5!', 'somepassworD!5!', 'somepassworD!5!@#', 'somepassworD!AFD#@$#3@DSAF@Q#$ADF@#$SDF@#$'];
```

Comment:
Codacy found an issue: [This line has a length of 161. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)